packages/docs/src/page-configs/ui-elements/data-table/examples/CRUD.vue

```diff
@@ -138,11 +139,28 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-  .table-example--slot {
-    th {
-      padding-top: 5px;
-      padding-bottom: 5px;
-      vertical-align: middle;
+  .table-example {
+    &__slot {
+      th {
+        padding-top: 5px;
+        padding-bottom: 5px;
+        vertical-align: middle;
+      }
+
+      .va-input {
+        --va-form-element-default-width: 80px;
+
+        width: 100%;
```

Comment:
Here we need to use some `d-block` css helper class like block or smth.
```suggestion
        display: block;
```